doc/xml/firewalld.policies.xml

```diff
@@ -0,0 +1,213 @@
+            </para>
+        </refsect2>
+
+        <refsect2 id="activepolicies">
+            <title>Active Policies</title>
+            <para>
+                Policies only become active if all of the following are true.
+            </para>
+            <itemizedlist>
+                <listitem>
+                    <para>
+                        The ingress zones list contain at least one regular zone
+                        or a single symbolic zone.
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>
+                        The egress zones list contain at least one regular zone
+                        or a single symbolic zone.
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>
+                        For non symbolic zones, the zone must be active. That
+                        is, it must have interfaces or sources assigned to it.
+                    </para>
+                </listitem>
+            </itemizedlist>
+            <para>
+                If the policy is not active then the policy has no effect.
+            </para>
+        </refsect2>
+
+        <refsect2 id="symboliczones">
+            <title>Symbolic Zones</title>
+            <para>
+                Regular zones are not enough to express every form of packet
+                filtering. For example there is no zone to represent traffic
+                flowing to or from the host running firewalld. As such, there
+                are some symbolic zones to fill these gaps. However, symbolic
+                zones are unique in that they're the only zone allowed in the
+                ingress or egress zone sets. For example, you cannot use
+                "public" and "HOST" in the ingress zones.
+            </para>
+
+            <para>
+                Symbolic zones:
+            </para>
+            <orderedlist>
+                <listitem>
+                    <para>HOST</para>
+                    <para>
+                        This symbolic zone is for traffic flowing to or from
+                        the host running firewalld. This corresponds to
+                        netfilter (iptables/nftables) chains INPUT and OUTPUT.
+                    </para>
+                    <itemizedlist>
+                        <listitem>
+                            <para>
+                                If used in the ingress zones list it will apply to
+                                traffic on the INPUT chain.
+                            </para>
+                        </listitem>
+                        <listitem>
+                            <para>
+                                If used in the egress zones list it will apply
+                                to traffic on the OUTPUT chain.
+                            </para>
+                        </listitem>
+                    </itemizedlist>
```

Comment:
Sorry for intervening, but I'm puzzled by this doc part. Is it correct? Intuitively it seems like "ingress zone `HOST`" would mean "outgoing traffic from the host". Also, as far as I understand, this code applies `ingress-zone HOST` rules to `OUTPUT`, and `egress-zone HOST` to `INPUT`:
https://github.com/firewalld/firewalld/blob/f5307039a5e28371553b3241ba10f04811f75a82/src/firewall/core/fw_policy.py#L1905-L1915

Reply:
Packets have an ingress and egress.
INPUT = ingress regular zone and egress HOST (e.g. public -> host)
OUTPUT = ingress HOST and egress regular zone (e.g. host -> public)

"HOST" is a symbolic zone. It's doesn't literally mean "the host".

Reply:
>INPUT = ingress regular zone and **egress** HOST

Yes, that's what I'm saying. Doesn't this mean that the doc is misleading? It says
>HOST ... If used in the **ingress** zones list it will apply to traffic on the INPUT chain.

Reply:
You are correct. Sorry. :)
I'll push this update:
```
diff --git a/doc/xml/firewalld.policies.xml b/doc/xml/firewalld.policies.xml
index 9633ddd78adf..1049b5ab6e9b 100644
--- a/doc/xml/firewalld.policies.xml
+++ b/doc/xml/firewalld.policies.xml
@@ -128,13 +128,13 @@
                     <itemizedlist>
                         <listitem>
                             <para>
-                                If used in the ingress zones list it will apply to
+                                If used in the egress zones list it will apply to
                                 traffic on the INPUT chain.
                             </para>
                         </listitem>
                         <listitem>
                             <para>
-                                If used in the egress zones list it will apply
+                                If used in the ingress zones list it will apply
                                 to traffic on the OUTPUT chain.
                             </para>
                         </listitem>
```

Reply:
Done. Thanks for reading the docs. :)

Reply:
Cool, thanks.